### PR TITLE
Add Installation Instructions for Arch Linux

### PR DESCRIPTION
**Pull request by mastertinner**
Add installation instructions for AUR on Arch Linux

This pull request makes the following changes:
* Adds the installation instructions for AUR on Arch Linux
* Bundle installation instructions with Homebrew ones

cc @bhamail / @DarthHater

### CODE REVIEW

**Comment by mastertinner** [comment]
I just signed the CLA. How can I retrigger the check?

File: README.md [hunk 1]
```diff
@@ -546,16 +570,16 @@ Adding new files? Get the license header correct with:
...
-         
-  2. Push the tag up:
-  
-         git push origin v1.0.x
-         
-  3. There is no step 3.
-          
+1. Checkout/pull the latest `main` branch, and create a new tag with the desired semantic version and a helpful note:
+
+   git tag -a v1.0.x -m "Helpful message in tag."
```
Comment (bhamail) [inline README.md:575]: While many of the formatting changes are an improvement, such changes do not appear to be related to the intent of the PR (Arch install instructions). Happy to include the formatting changes, as they do no harm, and appear to improve things.

However, these particular changes to the Release Steps appear to remove the commands from "code" blocks. Maybe you could revert this particular format change? (Or indent the commands to they remain "code" blocks?)

**Review by bhamail — APPROVED** [review]
Thanks for the PR! Looks great. One minor tweak suggested.

File: README.md (hunk 1 above)
Comment (mastertinner) [reply README.md]: Good point, @bhamail. Thanks! It's fixed now.

**Pull request merged**